girder/utility/server.py

```diff
@@ -125,24 +125,32 @@ def configureServer(test=False, plugins=None, curConfig=None):
         plugins = settings.get(constants.SettingKey.PLUGINS_ENABLED, default=())
 
     plugins = list(plugin_utilities.getToposortedPlugins(plugins, ignoreMissing=True))
+
+    # If the static route is a URL, leave it alone
+    if '://' in routeTable[constants.GIRDER_STATIC_ROUTE_ID]:
+        apiStaticRoot = routeTable[constants.GIRDER_STATIC_ROUTE_ID]
+        staticRoot = routeTable[constants.GIRDER_STATIC_ROUTE_ID]
+    else:
+        # Make the staticRoot relative to the api_root, if possible.  The api_root
+        # could be relative or absolute, but it needs to be in an absolute form for
+        # relpath to behave as expected.  We always expect the api_root to
+        # contain at least two components, but the reference from static needs to
+        # be from only the first component.
+        apiRootBase = os.path.split(os.path.join('/', curConfig['server']['api_root']))[0]
+        apiStaticRoot = os.path.relpath(routeTable[constants.GIRDER_STATIC_ROUTE_ID],
+                                        apiRootBase)
+        staticRoot = os.path.relpath(routeTable[constants.GIRDER_STATIC_ROUTE_ID],
```

Comment:
Yep, that's true if you're referring to the local fileystem, but in this case we know we are in an environment that exclusively supports forward slash separators, so using `posixpath` is more defensive in case we ever want to deploy on windows.